src/pages/en/utokyo_account/mfa/initial/index.mdx

```diff
@@ -145,7 +145,7 @@ If the remote-access environment (Citrix Workspace) of office work devices for a
 {:.mfa-initial-procedure}
 
 <div class="box">
-You have completed the initial setup procedures for MFA. <strong>It will take about 40 minutes for the MFA settings to be reflected in the system after these procedures</strong>, so if you want to use UTokyo VPN and other services in which MFA is required, please be patient and wait for a while.
+You have completed the initial setup procedures for MFA, <strong>but it will take about 40 minutes for the MFA settings to be reflected in the system after these procedures.</strong> If you want to use systems that do not allow you to sign in until the multi-factor authentication settings are completed, please be patient and wait for a while.
```

Comment:
```suggestion
You have completed the initial setup procedures for MFA, <strong>but it will take about 40 minutes for the MFA settings to be reflected in the system after these procedures</strong>. If you want to use systems that do not allow you to sign in until the multi-factor authentication settings are completed, please be patient and wait for a while.
```